tests/cds-rdm/test_full_migration.py

```diff
@@ -133,11 +137,41 @@ def orcid_id(record):
                 "name": "Mendoza, Diego",
                 "given_name": "Diego",
                 "family_name": "Mendoza",
-                "identifiers": [{"identifier": "2773374", "scheme": "lcds"}],
+                "identifiers": [
+                    {"identifier": "2773374", "scheme": "lcds"},
+                    {"identifier": "0009-0007-7638-4652", "scheme": "orcid"},
+                ],
             }
         }
     ]
 
+    name_from_db = NamesMetadata.query.filter_by(pid=orcid_name_data["id"]).one()
+    assert "identifiers" in name_from_db.json
+    orcid_identifier = orcid_name_data["identifiers"][0]
+    assert orcid_identifier in name_from_db.json["identifiers"]
+
+    user_identity = UserIdentity.query.filter_by(
+        id=name_from_db.internal_id, method="cern"
+    ).one()
+    assert user_identity is not None
+
+    remote_account = RemoteAccount.query.filter_by(user_id=user_identity.id_user).one()
+    assert hasattr(remote_account, "extra_data")
+    assert remote_account.extra_data == {
+        "migration": {
+            "source": "PEOPLE COLLECTION, PERSON_ID FOUND",
+            "note": "MIGRATED INACTIVE ACCOUNT",
```

Comment:
there are 2 other cases in the conditional statement when we are creating accounts, could you also cover them with a test? 
We will refactor the code of users creation for the next collection so we need to cover as much as possible to make sure refactoring will not break anything in the logic